website/data/community.json

```diff
@@ -17,11 +17,11 @@
       "description": "<p>ChartIQ is a powerful and flexible HTML5 charting library with millions of users worldwide. Written in JavaScript, it runs entirely within the browser. With state of the art integrations, including  third-party apps, educational components, analysis, and more, you can create efficient workflows and streamline the way to work.</p><p>ChartIQ works on any platform (mobile, web, desktop) and any framework (Angular, React).</p><p>Most companies have at least one financial charting library for each platform or app they target—web, C#, Java, Android, iOS, etc., so developers have multiple code bases to maintain. With ChartIQ, write your code once and use it everywhere.</p>"
     },
     {
-      "title": "Electron-FDC3",
+      "title": "FDC3 Sail",
```

Comment:
Note community page implementation alpha sorts enties before rendering so ordering doesn't matter here (but does in users.json)

Reply:
added code to sort so we don't have to worry about doing this manually.

Reply:
thank you!